lib/aruba/matchers/file/have_same_file_content.rb

```diff
@@ -43,4 +42,21 @@
   end
 end
 
-RSpec::Matchers.alias_matcher :a_file_with_same_content_like, :have_same_file_content_like
+RSpec::Matchers.alias_matcher :a_file_with_same_content_as, :have_same_file_content_as
+
+# Deprecated matchers
+module DeprecatedMatchers
```

Comment:
I think we can safely deprecate these in the 0.14.x series, and drop them entirely for 1.0.

Reply:
Sounds good to me. I'll rebase this when I get a moment.